spec/tezedge-openapi.json

```diff
@@ -0,0 +1,5293 @@
+{
+  "openapi": "3.0.0",
+  "info": {
```

Comment:
@mambisi @jurajselep 
please, make sure that information are correct and relevant (email, links) and not like `This is a sample server for a pet store.`

and copy the same information to all files